src/metabase/api/permissions.clj

```diff
@@ -83,23 +83,23 @@
   The optional `sandboxes` key contains a list of sandboxes that should be created or modified in conjunction with
   this permissions graph update. Since data sandboxing is an Enterprise Edition-only feature, a 402 (Payment Required)
   response will be returned if this key is present and the server is not running the Enterprise Edition, and/or the
-  `:sandboxes` feature flag is not present."
-  [:as {body :body}]
-  {body :map}
+  `:sandboxes` feature flag is not present.
+
+  If the skip-graph query param is truthy, then the graph will not be returned."
+  [:as {body :body
+        {skip-graph :skip-graph} :params}]
```

Comment:
It's _possible_ to write a malli schema for the params directly here, but it would have to be optional (since the query param is optional), so this seemed a bit simpler.